tests/e2e/resources/agentctl.conf

```diff
@@ -0,0 +1,16 @@
+use-tls: true
```

Comment:
this option is already outdated, isn't it?

Reply:
it is, thanks